db_setup.md

```diff
@@ -0,0 +1,13 @@
+## Setting up the database
+
+To set up the database credentials, run the following commands:
+
+`dotnet users-secrets init`
+
+`dotnet users-secrets set "Server" "YourServerName"`
+
+`dotnet users-secrets set "Database" "YourDatabaseName"`
+
+`dotnet users-secrets set "Uid" "YourUsername"`
+
+`dotnet users-secrets set "DbPassword" "YourPassword"`
```

Comment:
The command is `dotnet user-secrets`. Please change accordingly.

Reply:
Done